components/renku_data_services/session/models.py

```diff
@@ -99,8 +99,8 @@ class EnvironmentPatch:
     container_image: str | None = None
     default_url: str | None = None
     port: int | None = None
-    working_directory: PurePosixPath | None = None
-    mount_directory: PurePosixPath | None = None
+    working_directory: PurePosixPath | ResetType | None
+    mount_directory: PurePosixPath | ResetType | None
```

Comment:
suggestion: For consistency's sake it is better to keep the default `None` value on these just like all the other  fields.

Also we can remove `EnvironmentUpdate` above this. I know you did not touch. But I noticed it is not used anywhere at all. It is a leftover from some other PR where we should have got rid of it.

Reply:
Done in 8914a70067a9b6c7332d276020c64226e17c0b48.